src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -424,33 +424,9 @@
   <data name="ResponseCache_SpecifyDuration" xml:space="preserve">
     <value>If the '{0}' property is not set to true, '{1}' property must be specified.</value>
   </data>
-  <data name="ApiParameterSource_Body" xml:space="preserve">
-    <value>Body</value>
-  </data>
-  <data name="ApiParameterSource_Custom" xml:space="preserve">
-    <value>Custom</value>
-  </data>
-  <data name="ApiParameterSource_Header" xml:space="preserve">
-    <value>Header</value>
-  </data>
-  <data name="ApiParameterSource_Hidden" xml:space="preserve">
-    <value>Hidden</value>
-  </data>
-  <data name="ApiParameterSource_ModelBinding" xml:space="preserve">
-    <value>ModelBinding</value>
-  </data>
-  <data name="ApiParameterSource_Path" xml:space="preserve">
-    <value>Path</value>
-  </data>
-  <data name="ApiParameterSource_Query" xml:space="preserve">
-    <value>Query</value>
-  </data>
-  <data name="ApiParameterSource_Form" xml:space="preserve">
-    <value>Form</value>
-  </data>
   <data name="ApiExplorer_UnsupportedAction" xml:space="preserve">
     <value>The action '{0}' has ApiExplorer enabled, but is using conventional routing. Only actions which use attribute routing support ApiExplorer.</value>
-    </data>
+  </data>
```

Comment:
I still see the use of strings in the pr, what was the reason why it was interesting to add a resource earlier but is no longer the case?